### ISSUE TITLE
Add feedback/teammessage.txt for automated teamfeedback

### ISSUE THREAD

**Issue by gozzarda** [DOMjudge/domjudge#991]
https://github.com/DOMjudge/domjudge/blob/3f8c8b1f8d69c142ea90d34653c55a2c07d195c7/sql/files/examples/boolfind_cmp/run#L9

The above file and the manual from October 2017 mentioned the existence of a `teammessage.txt` feedback file that, presumably, would be shown to the team as submission feedback like `judgemessage.txt` is shown to the judges.

I have recently had cause to desire this functionality. It would in general be especially useful for practise contests, where you don't want to give the students all the answers, but giving them automated feedback to help them figure it out could be very valuable.

Strangely, I can't even find where these mentions were ever removed. What happened to this feature? Was it removed deliberately, or did it just get forgotten half way through implemenation? Can we have this feature again in the future?

**Comment by thijskh** [comment]
Hi! It took indeed some archive research but the documentation mention was removed in ce9c2c3e0458e3a98e29fbb105eafb78e7b19bb7. This is because it turned out that we documented that we supported it, but in fact we never have. So the documentation was brought up to date with reality: the feature never existed in DOMjudge.

Indeed from `boolfind_cmp/run` the mention should be removed for consistency, at least.

Of course it would be nice if DOMjudge could support this. For this a pull request would be most welcome.
